docs/docs/10.6-create-fragment.md

```diff
@@ -11,24 +11,21 @@ In most cases, you can use the `key` prop to specify keys on the elements you're
 That is, if you have a component such as:
 
 ```js
-var Swapper = React.createClass({
-  propTypes: {
-    // `leftChildren` and `rightChildren` can be a string, element, array, etc.
-    leftChildren: React.PropTypes.node,
-    rightChildren: React.PropTypes.node,
-
-    swapped: React.PropTypes.bool
-  },
-  render: function() {
-    var children;
-    if (this.props.swapped) {
-      children = [this.props.rightChildren, this.props.leftChildren];
-    } else {
-      children = [this.props.leftChildren, this.props.rightChildren];
-    }
-    return <div>{children}</div>;
+const Swapper  = ({props}) {
```

Comment:
@roganov is correct, but let's just use an es5 function for this component.  It is more familiar to newbies coming to React.